### DISCUSSION TITLE
What is your favorite anime?

### DISCUSSION THREAD

**Discussion by shumpe-m** [NAIST-SE/AttractingContributors#418] [General]
What is your favorite anime?

あなたの好きなアニメを教えてください

https://github.com/shunpe-hash/AttractingContributors-anime.git

**Comment by aru1702** [comment]
I like one piece, and lately I watch mystery theme like SPY x FAMILY and Grisaia animation

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
